migrations/20210616085921_indexes.js

```diff
@@ -0,0 +1,21 @@
+exports.up = function (knex) {
+  const promises = [];
+  promises.push(knex.schema.table('psychologists', (table) => {
+    table.index(['archived', 'state', 'active']);
```

Comment:
This would create only 1 index on the 3 columns together and not 3 indexes on each column. Is it what you expected?

Reply:
yes, because if you look at the query we are always searching with this 3 columns
so i think in that specific case a triple index behave more efficiently